cachito/paths.py

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import logging
+import shutil
+
+from pathlib import Path
+
+log = logging.getLogger(__name__)
+
+
+class RequestBundleDir(type(Path())):
+    """
+    Represents a directory tree for a request
```

Comment:
Sorry about these types of comments. I should really add `flake8-docstrings` to the linting done by tox to make it easier on contributors.
```suggestion
    Represents a directory tree for a request.
```